packages/web/src/utils/escapeArguments.ts

```diff
@@ -0,0 +1,7 @@
+export const escapeArguments = (args: string[]) => {
+  return args.map(arg => {
+    const escapedArg = arg.replace(/(["'`])/g, '\\$1');
+
+    return /\s/.test(escapedArg) ? `"${escapedArg}"` : escapedArg;
```

Comment:
`abc' def` will result in `"abc\' def"` (unnecessary escaping of `'`)

```suggestion
    return /\s/.test(escapedArg) ? `"${escapedArg.replace(/\\'/g, '\''}"` : escapedArg;
```
+ unit test